config/prod/dgutman-dsa-server-prod-ec2.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-prod-ec2"
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","443", "22"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "HTAN"
+  # The resource owner
+  OwnerEmail: "bruno.grande@sagebase.org"
```

Comment:
this email is tied to parkmycloud and allow the OwnerEmail to manage this instance in PMC.  if it's not set to dgutman then he will not have access to start/stop this instance.

Reply:
i'm not sure how that happened but it looks like dgutman has a sage jumpcloud account.

Reply:
We onboarding him as a contract worker